backend/app/services/launch_pad.rb

```diff
@@ -7,26 +7,20 @@ def initialize(study_id:, user_id:)
     @user_id = user_id
   end
 
-  def launch
+  def launch_url(preview: false)
     raise(LaunchError, 'This study is not open.') unless study.open?
```

Comment:
Shouldn't we allow researchers to preview studies that are not yet open?

Reply:
yes, I suppose we might as well allow that. shouldn't hurt anything and I can see where it'd be useful